cpp/include/cugraph/graph_view.hpp

```diff
@@ -679,6 +682,21 @@ class graph_view_t<vertex_t,
   edge_t count_self_loops(raft::handle_t const& handle) const;
   edge_t count_multi_edges(raft::handle_t const& handle) const;
 
+  void attach_mask_view(std::vector<graph_mask_view_t<vertex_t, edge_t>*>& graph_mask)
+  {
+    mask_ = std::make_optional<std::vector<graph_mask_view_t<vertex_t, edge_t>*>>(graph_mask);
+  }
+
+  std::optional<graph_mask_view_t<vertex_t, edge_t>> get_mask_view(size_t partition_idx = 0) const
+  {
+    graph_mask_view_t<vertex_t, edge_t>* mask = (*mask_)[partition_idx];
+    return mask_.has_value()
+             ? std::make_optional<graph_mask_view_t<vertex_t, edge_t>>(*(*mask_)[partition_idx])
+             : std::nullopt;
+  }
+
+  std::optional<vertex_t const*> local_sorted_unique_edge_src_begin() const;
```

Comment:
What's this? Why did you add this? It seems like this appeared here due to some erroneous merge conflict resolution.